.ci/azure-pipelines/tutorials.yaml

```diff
@@ -2,8 +2,8 @@ jobs:
   - job: tutorials
     displayName: Building Tutorials
     pool:
-      vmImage: 'Ubuntu 16.04'
-    container: env1604
+      vmImage: 'Ubuntu 20.04'
+    container: env1804
```

Comment:
Want to keep the container change in this one?

Reply:
The tutorials shouldn't run on an image we're no longer interested in, right?

Reply:
No, its just not fit for the headline of the PR :-)

Reply:
But should be moved to another PR?

Reply:
It's literally a 1 char change. Let me reword the title